runtime/j9vm/java11vmi.c

```diff
@@ -1815,4 +1815,11 @@ JVM_AreNestMates(JNIEnv *env, jclass clzOne, jclass clzTwo)
 	assert(!"JVM_AreNestMates unimplemented");
 	return JNI_FALSE;
 }
+#if J9VM_JCL_SE12
```

Comment:
Please create a new file for this change, its not appropriate for java11vmi.c

Reply:
Created a new file `javanextvmi.c` to contain this `JDK12` specific `JVM` method, pls have another look.